### DISCUSSION TITLE
Matching the light intensity and direction of foreground (pyracanny/product) and background (generated)

### DISCUSSION THREAD

**Discussion by himpundir** [lllyasviel/Fooocus#3252] [Q&A]
how can I generate a model face for a earring such that model face generated should have similar light direction and intensity as in the product png of earring (pyracanny)? Right now the face generated looks off in terms of light direction and intensity.
![image - 2024-07-08T131609 356](https://github.com/lllyasviel/Fooocus/assets/174606763/5450a94b-961c-465b-b858-3183cec1479a)
![GPGN-D0007399asd](https://github.com/lllyasviel/Fooocus/assets/174606763/5b8b0451-7ea8-4a00-91da-1b3cef6d24ff)

**Comment by mashb1t** [comment]
My proposal would be to add the earring to the image using Photoshop/Gimp/another image editing tool, then use upscale x2 to re-render the image to adjust the lighting in general.
